charts/monitor/values.yaml

```diff
@@ -14,6 +19,11 @@ influxdb:
   database: "kubernetes"
   user: "user"
   password: "password"
+  persistence:
+    enabled: false
+    storageClass: standard
+    accessMode: ReadWriteOnce
+    size: 20Gi
```

Comment:
I'm talking about the actual size of the volume claim. Some users may want smaller or bigger voluime claims.